src/jira.js

```diff
@@ -555,10 +555,17 @@ export default class JiraApi {
    * @name getVersions
    * @function
    * @param {string} project - A project key to get versions for
+   * @param {object} query - An object containing the query params
    */
-  getVersions(project) {
+  getVersions(project, query = {}) {
     return this.doRequest(this.makeRequestHeader(this.makeUri({
       pathname: `/project/${project}/versions`,
+      query: Object.keys(query).reduce((acc, key) => {
+        if (query[key]) {
```

Comment:
Let's pass `query` directly? It's not clear why only truthy values are allowed. Also the lib is not really responsible for cleaning the inputs.